cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -903,6 +903,62 @@ static wifi_config_t wifi_config_ap = {
 };
 #endif /* (defined(MCU_ESP8266) && !defined(MODULE_ESP_NOW)) || defined(MODULE_ESP_WIFI_AP) */
 
+
+#if (defined(MCU_ESP32) && defined(MODULE_ESP_WIFI_CSI))
+void __attribute__((weak)) esp_wifi_csi_rx_cb(void *ctx, wifi_csi_info_t *info)
+{
+    if (!info || !info->buf) {
+        ESP_WIFI_LOG_ERROR("ERROR: esp_wifi_csi_rx_cb missing info or buf");
+        return;
+    }
+    printf("{");
```

Comment:
Does it really make sense to embed everything (identifier and value) in `\"`? Furthermore, it is difficult for humans to read when everything is on one line without commas. Or is it just needed for postprocessing?

Reply:
I would assume there would be some post processing as there is lots of data there.  I can indent/newline it a bit better.

Reply:
> Does it really make sense to embed everything (identifier and value) in `\"`?

The question still remains, are all those double quotes required for the identifiers?